src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -141,10 +145,22 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             // Trigger a re-fetch of suppressions so the Roslyn settings are updated.
             await suppressionIssueStoreUpdater.UpdateAllServerSuppressionsAsync();
 
+            if (shareBinding)
+            {
+                progress?.Report(new MigrationProgress(0, 1, "Saving Shared Binding Config...", false));
+                SaveSharedBinding(oldBinding);
+            }
+
             progress?.Report(new MigrationProgress(0, 1, "Migration finished successfully!", false));
             logger.WriteLine(MigrationStrings.Process_Finished);
         }
 
+        private void SaveSharedBinding(BoundSonarQubeProject oldBinding)
+        {
+            var sharedBindingConfigModel = new SharedBindingConfigModel { Organization = oldBinding.Organization?.Key, ProjectKey = oldBinding.ProjectKey, Uri = oldBinding.ServerUri.ToString() };
+            _ = sharedBindingConfigProvider.SaveSharedBinding(sharedBindingConfigModel);
```

Comment:
Why ignore the result? Pretty sure Migration Window has progress output, could be one of the steps in that

Reply:
I thought about that but the rest of the code do not use this for error reporting and I don't think it would make sense. In case of an IO Error...etc we already log the issue so logging here would be redundant

Reply:
How would the user know they had an issue?

Reply:
Same as any other step: logs.